projects/graph/src/graph.py

```diff
@@ -6,4 +6,13 @@
 class Graph:
     """Represent a graph as a dictionary of vertices mapping labels to edges."""
     def __init__(self):
-        pass  # TODO
+        self.vertices = {}
+
+    def add_vertex(self, vertex):
+        self.vertices[vertex] = set()
+
+    def add_edge(self, vertex1, vertex2):
+        if vertex1 and vertex2 in self.vertices:
+            self.vertices[vertex1].add(vertex2)
+            self.vertices[vertex2].add(vertex1)
+      
```

Comment:
Please continue to work on the Stack and Queue classes, and also the BFT, DFT, BFS, DFS, and recursive methods in the `Graph` class.